dss/operations/checkout.py

```diff
@@ -0,0 +1,68 @@
+"""
+Tools for managing checkout buckets
+"""
+import typing
+import argparse
+import logging
+import os
+
+from dss import Replica
+from dss.operations import dispatch
+from dss import Config
+from cloud_blobstore import BlobNotFoundError
+from dss.api.bundles import get_bundle_manifest
+from dss.storage.hcablobstore import compose_blob_key
+
+from dss.storage.identifiers import BUNDLE_PREFIX
+
+logger = logging.getLogger(__name__)
+
+checkout = dispatch.target("checkout", help=__doc__)
+
+
+def verify_delete(handler, bucket, key):
+    logger.warning(f'attempting removal of key: {bucket}/{key}')
+    try:
+        handler.delete(bucket=bucket, key=key)
+        handler.get(bucket=bucket, key=key)
+    except BlobNotFoundError:
+        logger.warning(f'Success! unable to locate key {bucket}/{key} ')
+    else:
+        logger.warning(f'Unable to delete key: {bucket}/{key}')
+        exit(1)
+
+
+@checkout.action("remove_checkout",
+                 arguments={"--replica": dict(choices=[r.name for r in Replica], required=True),
+                            "--keys": dict(nargs="+", help="keys from checkout bucket to remove", required=False),
+                            "--bundle-fqid": dict(nargs="+", help="fqid checkout bucket to remove", required=False)})
+def remove_checkout(argv: typing.List[str], args: argparse.Namespace):
+    """
+    Remove a bundle from the checkout bucket
+    """
+    replica = Replica[args.replica]
+    handler = Config.get_blobstore_handle(replica)
+    bucket = replica.checkout_bucket
+    stage = os.getenv('DSS_DEPLOYMENT_STAGE')
+    logger.warning(f'Remove Checkout Started for stage: {stage}, bucket: {bucket}')
```

Comment:
This can be removed: The user will receive feedback about each object removed from the checkout bucket.

Reply:
done